beta/experience-platform-extension/tutorials/tutorial-2-swift-core-setup.md

```diff
@@ -0,0 +1,116 @@
+
+### Install the AEPCore SDK
+
+Add the following code to the `Podfile`
+
+```
+
+  pod 'AEPServices', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPCore', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPLifecycle', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPIdentity', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPSignal', :git => 'git@github.com:adobe/aepsdk-core-ios.git', :branch => 'main'
+  pod 'AEPRulesEngine', :git => 'git@github.com:adobe/aepsdk-rulesengine-ios.git', :branch => 'main'
+```
+
+Run `pod install`, now it should download all the dependencies and add them to the project.
+
+Open the project from `AEPSampleApp.xcworkspace`
+
+
+
+### Init SDK
+
+1. Open the AppDelegate.swift, add the following import statements.
+
+```
+import AEPCore
+import AEPLifecycle
+import AEPIdentity
+import AEPSignal
+```
+
+2. In the method ` func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool` , register `Lifecycle` , `Identity` and `Signal` extensions and start the SDK. Also call the `MobileCore.configureWith(appId:)` to configure the app id. 
+
+```
+
+        MobileCore.setLogLevel(level: .trace)
+			  MobileCore.registerExtensions([Lifecycle.self, Identity.self, Signal.self], {
+
+            // Use the App id assigned to this application via Adobe Launch
+            MobileCore.configureWith(appId: self.LAUNCH_ENVIRONMENT_FILE_ID)
+        })
+```
+
+3. In [Adobe Experience Platform Launch](https://experience.adobe.com/launch), go to the **Environments** tab in the mobile property created in the previous step (Configure the Adobe Experience Platform Mobile SDK) and click on the Development![img](https://firebasestorage.googleapis.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-Lf1Mc1caFdNCK_mBwhe%2F-Lf1N06T8hdv0-r5jPPN%2F-Lf1N3-ofPO9fLFT1edw%2Fscreen-shot-2018-10-18-at-11.22.17-am.png?generation=1558039279051937&alt=media)icon. Find the Environment File ID at the top and copy it.
+
+   Set the  `LAUNCH_ENVIRONMENT_FILE_ID` to the copied Environment File ID in the `AppDelegate` (iOS) class.
+
+4. Now run the app, you should be able to see the SDK logs being printed in the console.
+
+
+
+### Enable Lifecycle
+
+1. Open the file `SceneDelegate.swift`. 
+
+2. In the `sceneWillEnterForeground` method, add:
+
+   ```
+   MobileCore.lifecycleStart(additionalContextData: nil)
+   ```
+
+3. In the `sceneDidEnterBackground` method, add:
+
+   ```
+   MobileCore.lifecyclePause()
+   ```
+
+4. Now run the app, since we don't have a Analytics extension yet, you won't be see any request going out, but in the log, you can find Lifecycle event which contains the `LaunchEvent` and other lifecycle data.
```

Comment:
break up these thoughts into multiple sentences.  it's hard to follow along with all the commas in a run-on sentence  :)